pynamodb/connection/transaction.py

```diff
@@ -0,0 +1,180 @@
+from pynamodb.exceptions import GetError, DoesNotExist
+
+from pynamodb.connection.base import Connection
+from pynamodb.constants import (
+    TABLE_NAME, PROJECTION_EXPRESSION, UPDATE, GET, PUT, DELETE, TRANSACT_ITEMS, CLIENT_REQUEST_TOKEN, CONDITION_CHECK,
+    CONDITION_EXPRESSION, EXPRESSION_ATTRIBUTE_NAMES, EXPRESSION_ATTRIBUTE_VALUES, UPDATE_EXPRESSION,
+    RETURN_VALUES_ON_CONDITION_FAILURE, ITEM, KEY, RETURN_VALUES, RESPONSES
+)
+
+PUT = PUT.lower().capitalize()
+DELETE = DELETE.lower().capitalize()
+
+TRANSACT_ITEM_LIMIT = 10
```

Comment:
nit: maybe link the necessity of the magic number (https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/Limits.html#limits-dynamodb-transactions)